src/effects/Phaser.h

```diff
@@ -91,21 +91,21 @@ class EffectPhaser final : public StatefulPerTrackEffect
    EffectType GetType() const override;
    RealtimeSince RealtimeSupport() const override;
 
-   unsigned GetAudioInCount() const override;
-   unsigned GetAudioOutCount() const override;
-   bool ProcessInitialize(EffectSettings &settings, double sampleRate,
-      ChannelNames chanMap) override;
-   size_t ProcessBlock(EffectSettings &settings,
-      const float *const *inBlock, float *const *outBlock, size_t blockLen)
-      override;
-   bool RealtimeInitialize(EffectSettings &settings, double sampleRate)
-      override;
-   bool RealtimeAddProcessor(EffectSettings &settings,
-      unsigned numChannels, float sampleRate) override;
-   bool RealtimeFinalize(EffectSettings &settings) noexcept override;
-   size_t RealtimeProcess(size_t group,  EffectSettings &settings,
-      const float *const *inbuf, float *const *outbuf, size_t numSamples)
-      override;
+   // Temporarily needed
+   size_t ProcessBlock(EffectSettings& settings,
+      const float* const* inBlock, float* const* outBlock, size_t blockLen)
+      override
+   {
+      return 0;
+   }
+
+   // Temporarily needed
+   size_t RealtimeProcess(size_t group, EffectSettings& settings,
+      const float* const* inbuf, float* const* outbuf, size_t numSamples)
+      override
+   {
+      return 0;
+   }
```

Comment:
I don't like the unexplained "Temporarily needed" functions.

Instead, change the base class to PerTrackEffect, then you don't need these two overrides.

Also, declare `MakeInstance` as override.

Reply:
Ok, integrated that in step 3

Reply:
Ok, both integrated into step 3 now.